### PR TITLE
Added a short quicksort algorithm

### PR DESCRIPTION
**Pull request by gundeep77**
### **Describe your change:**

- [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: sorts/short_quicksort.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def QuickSort(A):
```
Comment (mateuszz0000) [inline sorts/short_quicksort.py:1]: What is `A`? Why CamelCase in python? Please add doctests, typehints,description then we will review logic.

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]

File: sorts/short_quicksort.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		for item in A:
+			if item<pivot:
+				lesser.append(item)
+			else:
+				greater.append(item)
+		return QuickSort(lesser) + [pivot] + QuickSort(greater)
+
+# A = [5, 1, 7, 2, 9, 4, 3]
+print("Enter the elements: ")
+x= [int(p) for p in input().split()]
```
Comment (cclauss) [inline sorts/short_quicksort.py:19]: This line will hang our continuous integration tests as discussed in CONTRIBUTING.md.

**Comment by cclauss** [comment]
How is this different / better than https://github.com/TheAlgorithms/Python/blob/master/sorts/quick_sort.py#L14 ?

File: sorts/short_quicksort.py (hunk 1 above)
Comment (gundeep77) [reply sorts/short_quicksort.py:1]: I am sorry, i think for to change the name of the formal parameter. I am not much experienced as of now, in a learning stage, didnt know about conventional python casing. Will review it and re-request.
Thanks

**Pull request closed without merging**